packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -400,7 +400,10 @@ class Uploader {
    */
   saveState (state) {
     if (!this.storage) return
-    this.storage.set(`${Uploader.STORAGE_PREFIX}:${this.token}`, jsonStringify(state))
+    // make sure the keys get cleaned up.
+    // https://github.com/transloadit/uppy/issues/3748
+    const keyExpirySec = 60 * 60 * 24
+    this.storage.sendCommand('SET', [`${Uploader.STORAGE_PREFIX}:${this.token}`, jsonStringify(state), 'EX', keyExpirySec])
```

Comment:
```suggestion
    const redisKey = `${Uploader.STORAGE_PREFIX}:${this.token}`
    this.storage.set(redisKey, jsonStringify(state), 'EX', keyExpirySec)
```

Reply:
.set() can handle the 'EX' parameters, too.

Reply:
Ok, initially I tried `this.storage.set(redisKey, jsonStringify(state), { EX: keyExpirySec })` but that failed so I went with `sendCommand` instead. But if you know that this works, then I trust you!